tuf/ngclient/_internal/requests_fetcher.py

```diff
@@ -141,7 +141,7 @@ def _chunks(
             )
 
         except urllib3.exceptions.ReadTimeoutError as e:
-            raise exceptions.SlowRetrievalError(str(e))
+            raise exceptions.DownloadError from e
```

Comment:
Reasons for this are good but now the documented exceptions (here and in Fetcher API) are incorrect... Hmm. Maybe you could make SlowRetrievals speed argument optional instead?

Reply:
or figure out another solution -- I didn't spend a lot of time on that suggestion

Reply:
Yep, optional seems to fit well: 7cbfd0e